content/html/concepts/semantic-html/terms/details/details.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: '<details>'
+Description: 'Encapsulated the <summary> element and any additional details which are only visible when the element is in an open state.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Semantic'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<details>` semantic HTML element encapsulated the <summary> element and any additional elements which are only visible when the element is in an open state.
```

Comment:
Let's write these entries in present-tense as opposed to past-tense. (e.g. ~encapsulated~ -> encapsulates)